backend/api/Services/MapService.cs

```diff
@@ -63,6 +63,12 @@ public async Task AssignMapToMission(MissionRun missionRun)
             var positions = new List<Position>();
             foreach (var task in missionRun.Tasks)
             {
+                if (!task.Inspections.Any())
+                {
+                    positions.Add(task.RobotPose.Position);
+                    continue;
```

Comment:
This will only happen if there are no inspections and therefor no inspectionTargets to add, but can be removed as the loop would be skipped anyways as its an empty list

Reply:
It was just to make it more clear that we either add the inspectiontarget or the position, not both